naptan_create/create_table.py

```diff
@@ -0,0 +1,62 @@
+from io import BytesIO
+import codecs
+import csv
+import boto3
+from zipfile import ZipFile
+from urllib import urlopen
+
+# dynamodb columns name
+COLUMNS = ['ATCOCode', 'NaptanCode', 'PlateCode', 'CleardownCode',
+           'CommonName', 'CommonNameLang', 'ShortCommonName', 'ShortCommonNameLang',
+           'Landmark', 'LandmarkLang', 'Street', 'StreetLang', 'Crossing',
+           'CrossingLang', 'Indicator', 'IndicatorLang', 'Bearing', 'NptgLocalityCode',
+           'LocalityName', 'ParentLocalityName', 'GrandParentLocalityName', 'Town',
+           'TownLang', 'Suburb', 'SuburbLang', 'LocalityCentre', 'GridType', 'Easting',
+           'Northing', 'Longitude', 'Latitude', 'StopType', 'BusStopType',
+           'TimingStatus', 'DefaultWaitTime', 'Notes', 'NotesLang',
+           'AdministrativeAreaCode', 'CreationDateTime', 'ModificationDateTime',
+           'RevisionNumber', 'Modification', 'Status']
+
+
+def lambda_handler():
+    url = urlopen("http://naptan.app.dft.gov.uk/DataRequest/Naptan.ashx?format=csv")
```

Comment:
Let's change the url to be in a constant var

Reply:
DONE